infrastructure/lib/constructs/compliant-resources.ts

```diff
@@ -31,3 +32,33 @@ export class SecureBucket extends cdk.Construct {
     this.bucket = bucket;
   }
 }
+
+export interface PITRTableProps {
+  /**
+   * The properties to configure the DynamoDB table
+   */
+  tableProps: dynamodb.TableProps;
+}
+/**
+ * Creates a DynamoDB table with point-in-time recover (PITR)
+ * enabled by default.
+ */
+export class PITRTable extends cdk.Construct {
```

Comment:
Love this construct! We want to consider whether `PITRTable` is the best name. Eventually we will want to consider things like KMS encryption at rest and a few other things. Is there maybe a more generic name that we could give this? I don't want to bikeshed too much but I don't want to have to refactor a name change later (which could result in resource replacement)

Reply:
I agree. I originally had `SecureTable` to go along with the buckets and compliant resources, but changed it since the only thing that was really done was PITR (it wasn't really "secure" at the moment). `SecureTable` or something else?

Reply:
yeah... it's not a secure table _yet_. but it will be when we add KMS CMK support... maybe `SecureTable` is good enough for now.

Reply:
Okay, I'll go with that.